alembic/operations/ops.py

```diff
@@ -1091,10 +1094,18 @@ def create_table(cls, operations, table_name, *columns, **kw):
 class DropTableOp(MigrateOperation):
     """Represent a drop table operation."""
 
-    def __init__(self, table_name, schema=None, table_kw=None, _reverse=None):
+    def __init__(self,
+                 table_name,
+                 schema=None,
+                 table_comment=None,
+                 table_info=None,
+                 table_kw=None,
+                 _reverse=None):
         self.table_name = table_name
         self.schema = schema
+        self.comment = table_comment
         self.table_kw = table_kw or {}
+        self.table_info = table_info or {}
```

Comment:
**mike bayer** (zzzeek) wrote:

implement as:

  self.info = table_kw.pop("info", None)

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/2989